### DISCUSSION TITLE
Refresh rate issue

### DISCUSSION THREAD

**Discussion by hijoh** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#690] [Q&A] [answered]
When I take photos, the display screen will have black blocks. How to solve the problem?

![image](https://github.com/user-attachments/assets/e6092fee-3e18-4e2e-89a6-b8eea74a4e11)

**Comment by board707** [comment]
It is not a problem, it is by design.
The picture on the panel is not stead, it is a result of fast switching blocks on and off.

**Reply by hijoh** [reply]
Listen to me, this experience is not good. It would be perfect if this problem could be improved！

**Reply by solhuebner** [reply]
This is how the panels are designed :)

**Reply by hijoh** [reply]
It seems that FPGA can be used to solve it.

**Reply by board707** [reply, marked as answer]
This can't be "solved" because any HUB75 panel is exposed in blocks.
It's up to you to consider it's wrong, but it's reality

Try to use a longer exposition time while taking a photo.
